### PR TITLE
changed tool and created workflow

### PR DESCRIPTION
**Pull request by polkhe**
workflow

### CODE REVIEW

File: topics/assembly/tutorials/general-introduction/tools.yaml [hunk 1]
```diff
@@ -2,15 +2,12 @@
 api_key: admin
 galaxy_instance: http://localhost:8080
 tools:
-- name: velvetoptimiser
+- name: package_velvet_1_2_10
```
Comment (bgruening) [inline topics/assembly/tutorials/general-introduction/tools.yaml]: That's wrong, a package_ does not contain any tool.

**Comment by polkhe** [comment]
@bgruening thanks, added

File: topics/assembly/tutorials/general-introduction/tools.yaml [hunk 2]
```diff
@@ -2,15 +2,15 @@
 api_key: admin
 galaxy_instance: http://localhost:8080
 tools:
+- name: package_velvet_1_2_10
```
Comment (bgruening) [inline topics/assembly/tutorials/general-introduction/tools.yaml]: This is not needed anymore?

Comment (polkhe) [inline topics/assembly/tutorials/general-introduction/tools.yaml]: tools - no, unless we care about tool_panel_section_label atm
workflow - yes

**Comment by bgruening** [comment]
ping @Slugger70

**Pull request merged**